### PR TITLE
Add en version

### PR DESCRIPTION
**Pull request by banyan**
based on @annettebr's review. (thanks!)

### CODE REVIEW

File: company-profile-en.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+![slide02](https://cloud.githubusercontent.com/assets/4939774/20208738/8173d800-a833-11e6-8e9d-3ec4a2af006e.png)
+
+- Our vision "Distributors of Wisdom" derived from Masa's personal experience in developing countries where he participated in volunteer projects.
+- Seeing educational inequalities with his own eyes, he founded Quipper to contribute to the disparities in education with internet technology and mobile devices.
+
+![slide03](https://cloud.githubusercontent.com/assets/14937062/20047069/e77fb7a6-a4f3-11e6-9c28-380fae0925d4.png)
+
+- We currently have 5 offices in London (HQ), Tokyo, Manila, Jakarta, and Mexico City, and a team working in Ho Chi Minh City.
+- The team has grown to over 500 employees, including our largest sales force of 300 staff across Indonesia in 12 local branches.
+- The product team consists of c. 50 members, which more than half are web engineers. (as of Sep 2016)
```
Comment (yuya-takeyama) [inline company-profile-en.md]: Just curious what this `c.` stands for.
(`弱` in Japanese?)

@annettebr ?

**Comment by yuya-takeyama** [comment]
cc @quipper/product-members

**Comment by eugiellimpin** [comment]
Very nice!

**Comment by bdesmero** [comment]
Thanks for this! Now we'll be able to use it for other countries too! :dancer:

**Pull request merged**